proposals/dao/fip_70.ts

```diff
@@ -0,0 +1,189 @@
+import hre, { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import { DeployUpgradeFunc, SetupUpgradeFunc, TeardownUpgradeFunc, ValidateUpgradeFunc } from '@custom-types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+import { expectApproxAbs, time } from '@test/helpers';
+
+chai.use(CBN(ethers.BigNumber));
+
+let uniswapEthBalanceBefore: any;
+let balancerFeiWethPool: any;
+
+/*
+FIP-70
+DEPLOY ACTIONS:
+- Create a WeightedBalancerPoolManagerBase to be admin of the DAO's WeightedPools
+- Create a FEI 30% / WETH 70% WeightedPool
+- Create a PCVDeposit for the new B-30FEI-70WETH Balancer pool owned by the DAO
+- Create a DelayedPCVMover to move the 2nd half of WETH/FEI liquidity
+*/
+
+export const deploy: DeployUpgradeFunc = async (deployAddress, addresses, logging = false) => {
+  // Create a WeightedBalancerPoolManager contract
+  const weightedBalancerPoolManagerBaseFactory = await ethers.getContractFactory('WeightedBalancerPoolManagerBase');
+  const weightedBalancerPoolManagerBase = await weightedBalancerPoolManagerBaseFactory.deploy(addresses.core);
+  await weightedBalancerPoolManagerBase.deployTransaction.wait();
+  logging && console.log('weightedBalancerPoolManagerBase: ', weightedBalancerPoolManagerBase.address);
+
+  // Create a new FEI/WETH pool
+  const weightedPoolTwoTokensFactory = await ethers.getContractAt(
+    'IWeightedPool2TokensFactory',
+    addresses.balancerWeightedPoolFactory
+  );
+  const tx: TransactionResponse = await weightedPoolTwoTokensFactory.create(
```

Comment:
will this pool have an oracle?

Reply:
yes, in the constructor I've set it to `true`